### PR TITLE
[FE] Move sources to `Integrations` menu entry

### PR DESCRIPTION
**Pull request by 3nvi**
## Background

In a process of improving our navigation menu for newcomers, we proposed to move all of the available "connectors" in a new menu entry named `Integrations` which will host all the 3rd-party services that Panther can connect to. 

This PR implements just that

## Screenshots

Screen Shot 2021-02-15 at 5 48 06 PM

## Changes

- Add new collapsable menu entry named `Integrations`
- Move all CS and LA sources under Integrations
- Move & rename Destinations entry under Integrations

## Testing

`npm run test`
- How did you test your change?

### CODE REVIEW

File: web/src/routes.tsx [hunk 1]
```diff
@@ -163,55 +148,91 @@ const PrimaryPageLayout: React.FunctionComponent = () => {
...
+                {/* ******************* INTEGRATIONS ***************************** */}
+                <Redirect
+                  exact
+                  from={urls.integrations.home()}
+                  to={urls.integrations.logSources.list()}
+                />
+                <Redirect
+                  exact
+                  from={`${urls.integrations.logSources.list()}:type`}
+                  to={urls.integrations.logSources.list()}
```
Comment (alexmylonas) [inline web/src/routes.tsx:179]: Hm what's this redirection protects from?

**Review by alexmylonas — APPROVED** [review]
Nice job Aggelos, this must have been tedious. 

LGTM! 💯

File: web/src/routes.tsx (hunk 1 above)
Comment (3nvi) [reply web/src/routes.tsx:179]: You added it a while ago :D I think it had to do with breadcrumb clicking

Comment (alexmylonas) [reply web/src/routes.tsx:179]: Can't recall right now, if you don't sync-to-release you could remove it.

**Review by kouknick — APPROVED** [review]
Great work @3nvi. Just a minor comment and we are good to go.
It's so awesome that we were proactive and didn't have all those Urls hardcoded, otherwise this would have been so much harder 😉 .

File: web/src/routes.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (kouknick) [reply web/src/routes.tsx:179]: I just saw your conversation above, you can ignore this one!

**Pull request merged**